.checkstyle

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file - it's an Eclipse-specific configuration file so it shouldn't be in the repo. I'd recommend not using `git add *` or `git add .` because you can wind up adding things you don't really want.